<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>644</width>
    <height>493</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>644</width>
    <height>493</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>644</width>
    <height>493</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Simple Blackjack (by Timofey Tsvirko)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="addCardButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>400</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Add card</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startGameButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>420</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopRoundButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>370</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="quitButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>330</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startAnotherRoundButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Another round</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="betLineEdit">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>200</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="makeBetButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>200</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Make a bet</string>
    </property>
   </widget>
   <widget class="QLabel" name="betLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Bet</string>
    </property>
   </widget>
   <widget class="QLabel" name="curretBetLabel">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>110</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="balanceLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Balance</string>
    </property>
   </widget>
   <widget class="QLabel" name="currentBalanceLabel">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QFrame" name="playerCardsFrame">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>230</y>
      <width>501</width>
      <height>191</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>488</width>
         <height>168</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="dealerCardsFrame">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>0</y>
      <width>501</width>
      <height>191</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>488</width>
         <height>168</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="hiddenCardLabel">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>110</width>
      <height>181</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>110</width>
      <height>181</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>110</width>
      <height>181</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(136, 138, 133)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="messageLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>311</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="dealerPointsLabel">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>200</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="playerPointsLabel">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>420</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>644</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
